.github/workflows/build.yml

```diff
@@ -94,31 +89,8 @@ jobs:
         if: matrix.php >= 8.0
         run: ./bin/behat -fprogress --strict --tags=@php8
 
-  static-analysis:
```

Comment:
Since we now run Psalm for all Symfony versions, we run it as part of the tests instead of running as a separate job

Reply:
In the current setup, you run Psalm only if test succeed, which prevents getting the full feedback. I'd rather keep a separate job. It makes the feedback more understandable and allows getting the full feedback without iterating until we manage to get a green CI.